### PR TITLE
iblinter 0.4.16

### PR DESCRIPTION
**Pull request by giginet**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by giginet** [comment]
@kateinoigakukun Sorry, I forgot to update Homebrew

File: iblinter.rb [hunk 1]
```diff
@@ -2,12 +2,12 @@ class Iblinter < Formula
...
-      :revision => "00d0f13a522708346d1578720545cdd469934e33"
+      :tag => "0.4.16",
+      :revision => "94dbc6cd5f305bc51c589727b64b5b9789e7a9dc"
   head "https://github.com/IBDecodable/IBLinter.git"
 
   bottle do
-    root_url "https://github.com/IBDecodable/IBLinter/releases/download/0.4.15"
+    root_url "https://github.com/IBDecodable/IBLinter/releases/download/0.4.16"
     cellar :any_skip_relocation
     sha256 "205f06fcef18ef56fcae24b21614261f24ca16f4fa154c31c39988d8ee89c73f" => :mojave
```
Comment (kateinoigakukun) [inline iblinter.rb]: I think we should also update bottle things 🍼  by `brew install iblinter --build-bottle` and `brew bottle`

Comment (giginet) [inline iblinter.rb]: I think this should be updated too

**Comment by giginet** [comment]
I made bottle and updated hash

https://github.com/IBDecodable/IBLinter/releases/tag/0.4.16

**Comment by giginet** [comment]
```
$ brew uninstall iblinter
$ brew untap ibdecodable/iblinter
$ brew tap ibdecodable/iblinter .
$ brew install iblinter
```

It works!

**Comment by kateinoigakukun** [comment]
Thanks!!!

**Pull request merged**